app/packs/_components/QuestionPackDetails.tsx

```diff
@@ -207,22 +216,31 @@ const PackDetails = ({ id, type }: { id: string; type: string }) => {
 
 export function QuestionPackDetails({
 	id,
-	type
+	type,
+	isReview = false
 }: {
 	id: string
 	type: string
+	isReview?: boolean
 }) {
 	const [isOpen, setIsOpen] = useState(false)
 	const isMobile = useIsMobile()
 
 	const buttonContent = (
 		<Button
-			className={cn(
-				'w-full bg-brand-blue-100 hover:bg-brand-blue-200 text-white'
-			)}
+			className={cn('w-full', !isReview && 'text-white')}
```

Comment:
This is not necessary, the text color is already set as part of the buttons variant
```suggestion
			className="w-full"
```